src/components/AdjustVaultSummary.tsx

```diff
@@ -110,22 +102,45 @@ const AdjustVaultSummary = () => {
   const canMakeOffer =
     !hasErrors &&
     isActive &&
-    (debtDeltaValue || collateralDeltaValue) &&
+    (debtInputAmount?.value || collateralInputAmount?.value) &&
     offerSigner?.isDappApproved;
 
+  const isButtonDisabled = !canMakeOffer;
+
+  const offerButtonLabel = useMemo(() => {
+    if (!offerSigner?.isDappApproved) {
+      assert(
+        isButtonDisabled,
+        'Button should be disabled when dapp is not enabled in wallet',
+      );
+      return 'Enable Dapp in Wallet';
```

Comment:
the disabling assertions help. it's still awkward to me that the button has an instruction. not a blocker.